vllm/attention/backends/torch_sdpa.py

```diff
@@ -13,9 +13,12 @@
 
 if is_cpu():
```

Comment:
Now that this attention backend is not shared with XPUs, I think we can remove `is_cpu`?

Reply:
removed.